test/cache_test.go

```diff
@@ -71,6 +74,11 @@ func testArtifactBuildCacheDir(ctx context.Context, t *testing.T, cfg targetConf
 		if c.GoBuild != nil {
 			return "${GOCACHE}"
 		}
+		if c.Bazel != nil {
+			// There is no good way to determine the bazel cache dir
+			// So just hardcode this for now
+			return "/tmp/dalec/bazel-local-cache"
```

Comment:
not sure if better but maybe we could a helper function in `helpers.go` or other place that resolves to this value? so can be set one and reused also in `cache.go`?

Reply:
I honestly just didn't want to expose it as a public API.